pipeline/README.md

```diff
@@ -98,18 +93,15 @@ implement a class that follows the `Tokenizer` interface from
 
 
 ### Running a Simple Command-Line Test
-Assuming you downloaded the pipeline package as a zip from the
-Cogcomp Cognitive Computation Group's web site. 
-
-The standard distribution for this package puts dependencies in `lib/`;
-the parser model in `data/`; and the config file in `config/`. There are
-two sample scripts that are provided to test that the
-pipeline works after you have downloaded
-it. `scripts/runPreprocessor.sh` takes as arguments a configuration file
+NOTE: These commands assume you ran `mvn install` and `mvn dependency:copy-dependencies`,
+which create the pipeline binary in `target/` and copies all dependency jars into 
+`target/dependency`.  
+Two sample scripts are provided to test that the pipeline works after you have downloaded
+it. `scripts/runPipelineOnDataset.sh` takes as arguments a configuration file
 and a text file; it processes the text file according to the
 properties set in the config file, and writes output to STDOUT.
 scripts/testPreprocessor.sh is a self-contained script that calls
```

Comment:
Minor: add single quotes?  `scripts/testPreprocessor.sh` is a self-contained ....